src/map/clif.c

```diff
@@ -18572,7 +18578,9 @@ void clif_defaults(void) {
 	clif->mvp_noitem = clif_mvp_noitem;
 	clif->changed_dir = clif_changed_dir;
 	clif->charnameack = clif_charnameack;
+#ifdef ENABLE_MONSTER_HP_BAR
```

Comment:
Yes, this must be removed, because otherwise, if a plugin tries to use one of these functions, it'll crash.

- There should never be any ifdef inside the `xxx_defaults` function for an interface
- There should never be any ifdef inside the `struct xxx_interface` definition